testsuite/tests/multicluster/test_simple_strategy.py

```diff
@@ -26,7 +26,9 @@ def test_simple_strategy(client, hostname, gateway, gateway2):
 
     dns_ip1 = dns.resolver.resolve(hostname.hostname)[0].address
     dns_ip2 = dns.resolver.resolve(hostname.hostname)[0].address
+    dns_ip3 = dns.resolver.resolve(hostname.hostname)[0].address
     assert dns_ip1 != dns_ip2, "Simple routing strategy should return IPs in a round-robin fashion"
+    assert dns_ip1 == dns_ip3, "Simple routing strategy should return IPs in a round-robin fashion"
```

Comment:
What about resolving many addresses (10, 50, 100 ??) and verifying if the round-robin is working on that?